en_us.lang

```diff
@@ -367,4 +367,4 @@ gui.xaero_entity_radar_type=Type:
 gui.xaero_entity_radar_color=Color:
 gui.xaero_entity_radar_visiblity=Visibility:
 gui.xaero_entity_radar_names=Names:
-gui.xaero_box_entity_radar_names=Render entity names under the dots. \n \n Option "Player List" - render while the "List Players" key binding is pressed (TAB by default).
\ No newline at end of file
+gui.xaero_box_entity_radar_names=Render entity names under the dots. \n \n Option "Player List" - render while the "List Players" key binding is pressed (TAB by default).
```

Comment:
For consistency reasons we should choose only one variant and use it in all places. Which one do you like the most?
* key bind(s)
* keybind(s)
* key binding(s)
* keybinding(s)